src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -195,11 +195,11 @@ namespace
     {
         { IDS_RENDERERPARAMS_FILTER_TYPE_1,             L"Blackman-Harris" },
         { IDS_RENDERERPARAMS_FILTER_TYPE_2,             L"Box" },
-        { IDS_RENDERERPARAMS_FILTER_TYPE_3,             L"Catmull-Rom Spline" },
-        { IDS_RENDERERPARAMS_FILTER_TYPE_4,             L"Cubic B-spline" },
+        { IDS_RENDERERPARAMS_FILTER_TYPE_3,             L" Catmull-Rom Spline - Legacy" },
```

Comment:
One space indentation is deliberate I suppose?

Reply:
It is intentional. My idea was to have the legacy filter entries slightly offset in the selection menu (see screenshot above) to make it visually more distinct.